panther_analysis_tool/main.py

```diff
@@ -397,6 +397,7 @@ def update_schemas(args: argparse.Namespace) -> Tuple[int, str]:
     # this must be called prior to setting up the client
     if args.aws_profile is not None:
         logging.info("Using AWS profile: %s", args.aws_profile)
+        session = boto3.Session(profile_name=args.aws_profile)
```

Comment:
We already do this for `upload_analysis`, but not here in `update_schemas`. Maybe we should do this as part of setting everything up so its always in place for any function that gets called?

Reply:
maybe a wrapper like `get_client` or something

Reply:
I added a `get_client` wrapper function llike you suggested.